src/ansys/fluent/core/scheduler/load_machines.py

```diff
@@ -118,21 +118,18 @@ def _parse_host_info(host_info):
         A list of dictionaries formatted as:
         {'machine-name' : ###, 'core-count' : ###}
     """
-    if (
-        (":" in host_info or "," in host_info)
-        and not "\\" in host_info
-        and not "/" in host_info
-    ):
-        # Filenames generally shouldn't have ':',
-        # so assume it's a string list and parse accordingly
-        sMod = 1 if host_info[0] == "[" else 0
-        sBeg = sMod
-        sEnd = len(host_info) - sMod
-        machine_data = host_info[sBeg:sEnd].split(",")
-    else:
+    print(host_info)
```

Comment:
Use `LOG.ino`

Reply:
Print line was a mistake I forgot to remove it. Thank you for catching it.

Reply:
Yeah, if you can remove it that would be great.